pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "ml-compiler-opt"
+description = "Tooling for ML in LLVM"
+readme = "README.md"
+requires-python = ">=3.8"
```

Comment:
can this say <=3.10 as well? Not relying much on py backcompat.

Reply:
Updated it to say `<3.11`. `<=3.10` doesn't allow for any of the `3.10.x` minor releases.